.github/workflows/test.yml

```diff
@@ -17,3 +17,15 @@ jobs:
       uses: actions/checkout@v2
     - name: Test
       run: go test -v ./...
+
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16.x
```

Comment:
Is there any way to say `latest`?

Reply:
hmm good question :D

Reply:
I looked at actions/setup... it does not look like it, oh well.